src/displayapp/screens/settings/Settings.h

```diff
@@ -20,6 +20,7 @@ namespace Pinetime {
         bool OnTouchEvent(Pinetime::Applications::TouchEvents event) override;
 
       private:
+        DisplayApp* app;
```

Comment:
I think this reference is unused and can be removed, and later also the passed variable from the constructor can be removed

Reply:
This is passed to the List class, for calling `app->StartApp()`

Reply:
whoops, overlooked that